java/test/jmri/jmrix/AbstractThrottleTest.java

```diff
@@ -1560,7 +1560,7 @@ public void testGetRosterEntry() {
     @Test
     public void testGetSpeed_float() {
         Assert.assertEquals("Full Speed", 127, instance.intSpeed(1.0F));
-        float incre = 0.007874016f;
+        float incre = 1.0f / 126.0f;
```

Comment:
This change causes CI to fail on line 1570.

Reply:
Yes.

PR #9464 will cause CI to fail. This PR will fix CI after applying #9464.

And I think afterwards we'll have the correct behavior. But that's not my call to make.

Heiko

Reply:
If so, this fix needs to be in #9464. We don't merge PRs that fail CI.

Reply:
OK.